torch/_inductor/kernel/mm.py

```diff
@@ -133,14 +133,29 @@ def tuned_mm(mat1, mat2, *, layout=None):
             )
 
     if m * n != 0 and use_cutlass_template(layout):
-        cutlass_template = CUTLASSGemmTemplate([mat1, mat2], layout, alpha=1, beta=0)
+        cutlass_template = CUTLASSGemmTemplate(
```

Comment:
The lowering required a minor change, in that we differentiate between those ops which support evt fusion and those which don't. Each cutlass_template can still have multiple ops, but make collective fusion decisions just based on the property of whether the op supports evt or not.